website/docs/kubernetes/port-forwarding.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 13
+title: Configuring port forwarding
+description: Configuring port forwarding for a Kubernetes pod or service
+keywords: [podman desktop, podman, port forwarding, objects, kubernetes]
+tags: [Configuring-port-forwarding, port-forwarding]
+---
+
+# Configuring port forwarding
+
+Within a Kubernetes cluster, you can access an application by its internal IP address. But, if you want to access the application from your local machine, use the port forwarding feature. Using UI, you can forward a local port to a port on the pod that runs your application. This way you can interact with the application running in a Kubernetes cluster from your local machine for debugging and testing purposes.
+
+You can use the port forwarding feature for the pods and services running on a Kubernetes cluster. Also, you can perform port forwarding for any exposed ports.
+
+#### Prerequisites
+
+Make sure you have:
+- A running Podman machine.
+- A running Kubernetes cluster.
```

Comment:
Might be interesting to link to other part of the documentation such as 

- A running Podman machine. => `/docs/podman/creating-a-podman-machine`
- A running Kubernetes cluster. => `/docs/kind/creating-a-kind-cluster`